README.md

```diff
@@ -1,12 +1,40 @@
-# Template project
-
-> Template for [Axone](https://axone.xyz) opensource projects.
+# Hub backend
 
 [![lint](https://img.shields.io/github/actions/workflow/status/axone-protocol/template-oss/lint.yml?branch=main&label=lint&style=for-the-badge&logo=github)](https://github.com/axone-protocol/template-oss/actions/workflows/lint.yml)
 [![conventional commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg?style=for-the-badge&logo=conventionalcommits)](https://conventionalcommits.org)
 [![contributor covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg?style=for-the-badge)](https://github.com/axone-protocol/.github/blob/main/CODE_OF_CONDUCT.md)
 [![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg?style=for-the-badge)](https://opensource.org/licenses/BSD-3-Clause)
 
+> Official Axone Hub repository.
+
+## Prerequisites
+
+Make sure you have the following installed on your machine:
+- [Node.js](https://nodejs.org/) (v18.20.0)
+- [npm](https://www.npmjs.com/) (comes with Node.js) or [Yarn](https://yarnpkg.com/) (~1.22.19)
+
+## Installation
+
+1. **Clone the repository:**
+   ```bash
+   git clone https://github.com/axone-protocol/hub-backend.git
+2. **Install dependencies:**
+    ```bash
+    npm install
```

Comment:
I woudn't encourage mixing the use of `npm` and `yarn` as they use different locking file formats to manage dependencies. So I suggest only mentioning `yarn` as the only package manager.